src/components/offer-card/OfferCard.tsx

```diff
@@ -0,0 +1,131 @@
+import { Avatar, Box, Typography, Chip, Stack } from '@mui/material'
+import { FC } from 'react'
+import { styles } from '~/components/offer-card/OfferCard.styles'
+import AppOfferButtons from './AppOfferButtons'
+import TutorRating from './TutorRating'
+import PublicOutlinedIcon from '@mui/icons-material/PublicOutlined'
+import SaveFavoriteButton from './SaveFavoriteButton'
+
+interface OfferCardProps {
+  avatar?: string
+  rating?: number
+  author: string
+  subjects?: [] | string[]
```

Comment:
An empty array can be array of string, do not create additional type with only empty array. 
```suggestion
  subjects?: string[]
```